src/Servers/Kestrel/test/FunctionalTests/Http2/ShutdownTests.cs

```diff
@@ -43,8 +43,6 @@ public ShutdownTests()
 
         [CollectDump]
         [ConditionalFact]
-        [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/9985", Queues = "Fedora.28.Amd64;Fedora.28.Amd64.Open")]
```

Comment:
Removing this, super old, this issue is closed, and unclear what the issue was anyways.
https://github.com/dotnet/aspnetcore-internal/issues/2481